### PR TITLE
Readme edits

### PR DESCRIPTION
**Pull request by flo39skmgw**
Content for human.

### CODE REVIEW

File: hello1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+hello1
```
Comment (flo39skmgw) [inline hello1.txt]: Why Fix??

File: hello2.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+hello2
```
Comment (flo39skmgw) [inline hello2.txt]: Approve

**Review by flo39skmgw — COMMENTED** [review]
Approve

File: README.md [hunk 3]
```diff
@@ -1,2 +1,4 @@
 # hello-world
 Just another repository
+
+Hi humans !!
```
Comment (n9829bjcp4) [inline README.md]:
```suggestion
Hi humans !
```

**Review by n9829bjcp4 — CHANGES_REQUESTED** [review]
Please Revers

File: README.md [hunk 4]
```diff
@@ -1,2 +1,4 @@
 # hello-world
 Just another repository
```
Comment (n9829bjcp4) [inline README.md:2]:
```suggestion
Just another repository !!
```
IMO

File: hello1.txt (hunk 1 above)
Comment (n9829bjcp4) [inline hello1.txt]:
```suggestion
hello1 !!
```

File: hello2.txt (hunk 2 above)
Comment (n9829bjcp4) [inline hello2.txt]:
```suggestion
hello2 !!
```

**Review by n9829bjcp4 — CHANGES_REQUESTED** [review]
Please fix

**Review by n9829bjcp4 — APPROVED** [review]

**Review by n9829bjcp4 — APPROVED** [review]

**Review by n9829bjcp4 — COMMENTED** [review]
Nice

**Review by n9829bjcp4 — CHANGES_REQUESTED** [review]
No

File: hello2.txt [hunk 5]
```diff
@@ -0,0 +1 @@
+hello2 !!
```
Comment (n9829bjcp4) [inline hello2.txt]:
```suggestion
hello2 !!!
```

**Review by n9829bjcp4 — CHANGES_REQUESTED** [review]

**Review by n9829bjcp4 — APPROVED** [review]

**Review by n9829bjcp4 — CHANGES_REQUESTED** [review]
Fix1

**Review by n9829bjcp4 — CHANGES_REQUESTED** [review]
Fix2

**Review by n9829bjcp4 — APPROVED** [review]

File: hello1.txt [hunk 6]
```diff
@@ -0,0 +1 @@
+hello1 !!
```
Comment (n9829bjcp4) [inline hello1.txt]:
```suggestion
hello1 !
```

File: hello2.txt [hunk 7]
```diff
@@ -0,0 +1 @@
+hello2 !!!
```
Comment (n9829bjcp4) [inline hello2.txt]:
```suggestion
hello2 !
```

**Review by n9829bjcp4 — CHANGES_REQUESTED** [review]

**Review by n9829bjcp4 — DISMISSED** [review]

**Review by n9829bjcp4 — APPROVED** [review]

**Pull request merged**